examples/resources/cdp_operational_database/resource.tf

```diff
@@ -28,6 +28,6 @@ resource "cdp_operational_database" "opdb" {
   // scale_type   = "MICRO" // valid options are "MICRO","LIGHT","HEAVY"
   // storage_type = "HDFS"  // valid options are "CLOUD_WITH_EPHEMERAL","CLOUD","HDFS"
 
-  disable_multi_az = false
+  // disable_multi_az = false
```

Comment:
please either set it to true or remove it, because this commented line is also in the - generated - documentation